geom/alg_convex_hull_test.go

```diff
@@ -159,7 +159,7 @@ func TestConvexHull(t *testing.T) {
 		{
 			// there are collinear points",
 			input:  "MULTIPOINT(7 7,8 5,7 2,6 5,5 5,4 6,4 2,3 7,2 1,14 9)",
-			output: "POLYGON((2 1,7 2,14 9,7 7,3 7,2 1))",
```

Comment:
Good catch :) The test data here was definitely wrong.

<img width="734" alt="image" src="https://user-images.githubusercontent.com/5352655/62493304-199bac00-b814-11e9-89d6-bea9de1ddd70.png">